admin-frontend/src/components/checkIn/checkIn.js

```diff
@@ -59,6 +59,10 @@ const CheckInView = (props) => {
     }
 
   },[])
+  // reload the page in 10 minutes to fix qr reader stopping scanning after a period of time
+  useEffect(() => {
+    setTimeout(() => window.location.reload(), 600000)
```

Comment:
Move setTimeout to var and clear it so it doesnt cause bugs.
<img width="425" alt="Screenshot 2020-11-27 at 17 53 19" src="https://user-images.githubusercontent.com/32368461/100466351-7130eb80-30d9-11eb-801d-d2c024a91981.png">